src/main/webapp/js/messages-loader.js

```diff
@@ -45,6 +45,36 @@ function buildMessageDiv(message){
   return messageDiv;
 }
 
+function submitTransReq() {
+  const url = '/feed';
+  const language = document.getElementById('language').value;
+  const messageContainer = document.getElementById('message-container');
+  messageContainer.innerText = 'Loading...';
+
+  const params = new URLSearchParams();
```

Comment:
Also, even though this code works on my chrome, my safari browser at version 12.1.1 doesn't seem to send the params properly, i.e. the server cannot find the language field upon post request. It seems that safari version >=12 should support the URLSearchParams syntax so I am not sure why. Please advise.

Reply:
The client side just produces the URLSearchParams() with a language field and string. When tested on chrome, the server side is able to pick up the params in the request object and find the language string correctly. When tested on Safari, the server side simply gets null when trying to retrieve language value from the request object.

Reply:
Hmm I am not sure about this one. I'd have to manually test this. Chrome is better anyway d:

Reply:
> Chrome is better anyway d:
I LOL-ed